app/models/speaker.rb

```diff
@@ -22,7 +22,7 @@ class Speaker < ApplicationRecord
 
   validates :slug, uniqueness: true
 
-  [:name, :bio, :job_title, :github_url, :twitter_url, :linkedin_url, :image].each do |attr|
+  [:name, :bio, :job_title, :github_url, :twitter_url, :linkedin_url, :image, :uuid].each do |attr|
     delegate attr, "#{attr}=", to: :profile, allow_nil: true
```

Comment:
Should we allow `uuid` to be `nil`?

Reply:
this allow nil is just for the delegation, uuid is validated in profile model